server/app.js

```diff
@@ -59,4 +54,10 @@ app.use(function (err, req, res, next) {
 	res.json({ error: err });
 });
 
+const PORT = 8084;
```

Comment:
Can you please tell us why did you make this change? We have server running on PORT 3001 actually... there is the exact same code written in server -> bin -> www ...

Reply:
If your PORT 3001 is consumed by some other application, you can add it in your .env `PORT=8084` if that works!!

Reply:
Oh! I just made that so I had confirmation that the server was running initially, 8084 was just an arbitrary port I was using on my machine. I am used to putting a bunch of the logic that is in the bin/www. right in the app.js file, and for some reason the console.log( ) statement in the bin/www. wasn't triggering upon running the server file! I changed it to use the .env port variable if and kept the console.log statement in app.js if that's okay for everyone?

Reply:
No problem on my end